### DISCUSSION TITLE
Feature Request - Don't activate File Explorer from "Create new note from template"

### DISCUSSION THREAD

**Discussion by pdelre** [chrisgurney/obsidian-note-toolbar#203] [Ideas]
Thank you for adding support for creating notes from Templater, it was something I was long looking forward towards. 

Could the logic to reveal the file in the explorer be made optional? TBH, it doesn't work in my workflow which keeps that panel hidden. What I'm using today is Meta Bind's Buttons which [also calls `create_new_note_from_template`](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/blob/2ee470bd886088327a16819db9e248d45aa6c6ed/packages/obsidian/src/ObsidianInternalAPI.ts#L225) but then the cursor is placed in the filename, which can use the keyboard to `tab` into the document body.

https://github.com/chrisgurney/obsidian-note-toolbar/blob/b5ccbaa0ab1526525a5f1216312b67ba6a0e1af8/src/Adapters/TemplaterAdapter.ts#L144-L147

**Comment by chrisgurney** [comment]
Thank you for the request @pdelre 

Let me look into that, and what the alternatives might be.

**Comment by chrisgurney** [comment]
@pdelre I've released [`1.16-beta-06`](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.16-beta-06), which lets you optionally choose a command to execute after creating the new file. It's blank by default, so the `reveal-active-file` command is no longer executed unless it's explicitly set to do so. (Though as mentioned I couldn't get this to work reliably in my own testing, while other commands seem to work fine.)
